src/pages/style-guide.md

```diff
@@ -0,0 +1,556 @@
+
+- For **other writing**, occasional exclamation marks are fine
+  - They can help maintain a friendly, casual tone
+
+    | ✅                                                    |
+    | ---------------------------------------------------- |
+    | keep your data AI-ready from collection to delivery! |
+
+### Emojis ❗️
+- For most **documentation**, emojis can be used sparingly, to provide information
+  - Emojis must not be used for decoration
+
+    | ✅                    | ❌                                    |
+    | -------------------- | ------------------------------------ |
+    | ✅ Geolocation entity | 🏔️ “Glass-box” technical architecture |
+
+
+- For **release notes** and **API documentation**, no emojis at all
+
+- For **blog posts** and the **main website**, occasional emojis are fine
+  - They can help maintain a friendly, casual tone
+
+    | ✅                                              |
+    | ---------------------------------------------- |
+    | 🎉 Reduce your data loading costs by up to 80%. |
+
+### Diagrams and images
+- Adding diagrams and images is encouraged
+- Diagrams and images should have white backgrounds
+- For images, use `.webp` or `.png` format
+- For **documentation** diagrams, [Mermaid](http://mermaid.js.org/#/) code is recommended as they adjust automatically
+
+## Inclusive language
+### Allowlist and denylist
+- Use allowlist and denylist instead of the older white/blacklist terminology
+
+    | ✅                         | ❌                         |
+    | ------------------------- | ------------------------- |
+    | allowed form elements     | whitelisted form elements |
+    | allowlisted form elements | whitelisted form elements |
+
+### Gendered language
+- Avoid gendered language
+- Use plural, or the gender-neutral "they" third-person pronoun
+
+  | ✅                  | ❌                   |
+  | ------------------ | ------------------- |
+  | users will need to | he will need to     |
+  | they will need to  | he/she will need to |
+  | five person-hours  | five man-hours      |
+
+## General technical terms
+### Acronyms
+- Acronyms should be in capitals
+
+    | ✅    |
+    | ---- |
+    | HTTP |
+    | CDI  |
+    | CLI  |
+    | API  |
+    | SQL  |
+    | SDK  |
+    | URL  |
+
+- Some can be pluralized with an "s", but not all
+
+  | ✅            | ❌     |
+  | ------------ | ----- |
+  | JSON         | JSONs |
+  | JSON schemas | JSONs |
+  | APIs         |       |
+  | SDKs         |       |
+  | URLs         |       |
+
+- For the **documentation** website, we have a docs plugin that adds a dotted line and a tooltip explanation to acronyms, e.g. hovering over "BDP" will show that it stands for "Behavioral Data Platform": add new acronym definitions to the `src/remark/abbreviations.js` file to enable this behavior
+- (Technically, these are all initialisms, not acronyms)
+
+### Capitalization
+- Third-party products should be capitalized as appropriate
```

Comment:
Minor suggestion - not just third-party products, ours too!

Might we add Enrich to this list - which comes with the benefit of clarifying a small piece of nuance in the next example (enrichment is a process that shouldn't be capitalised, Enrich is a product that should - if I have it right).

IMO there's nothing wrong at all with how it's written as-is, I just thought I'd share the idea :)

Reply:
Oh, I continued reading and there's a snowplow specific bit. So perhaps this comment is moot - apologies for the noise!

Reply:
I'll move the Snowplow bit up so it's more obvious